dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2/src/org/apache/cxf/transport/servlet/servicelist/FormattedServiceListWriter.java

```diff
@@ -0,0 +1,253 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
What changed here?

Reply:
StringEscapeUtils.escapeHtml4(...) is added to this class to increase security.

Reply:
Add Liberty change comment on change